### PR TITLE
Add git ignore

### PR DESCRIPTION
**Pull request by LarryIVC**
# Hello Microverse Tasks
Is the result of having carried out all the tasks of the `Hello Microverse` project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,145 @@
...
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
```
Comment (shella12) [inline README.md]: - [ ] I can see 👀  that you are using the Microverse README template to document your project in your README 💯 . But there are some sections missing. Please ☺️ refer to [this template](https://github.com/microverseinc/readme-template) and make sure all of the sections are added to your README. The only section that is OPTIONAL is the `FAQ` section. You can always skip 🏃‍♂️  it if you want to. Remember 💭  to customize the template to your project as well.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,145 @@
...
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Client PC</a></li>
+  </ul>
+</details>
+
```
Comment (shella12) [inline README.md]: - [x] Nice 👏  👍  you have added the Tech Stack of your project into README. But there are a few 🤌 misunderstandings 😅 I would like to clarify for you.

-   Visual studios is not a Client its a tool.
-    Client PC is not a Server in your case

> A client is a program that uses services that other programs provide. The programs that provide the services are called servers. The client makes a request for a service, and a server performs that service.

I suggest using `HTML/CSS` for your tech stack in this Project

File: README.md [hunk 3]
```diff
@@ -1,2 +1,145 @@
...
+### Key Features <a name="key-features"></a>
+
+> Tool used on Microverse
+
+- **[Git]**
+- **[GitHub]**
+- **[Linters]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [ ] Key 🔑  features in the Microverse template are for the main features of your project 
> A feature is a unit of the functionality of a software system that satisfies a requirement, represents a design decision, and provides a potential configuration option. 

Git, Github, linters are tools used in the project. They can’t be added to the key 🔑  features section. 
🤓 I suggest using the real features of the project for this section. For Git, Github, linters you can add tools section into your README.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,145 @@
...
+<!--
+Example:
+
+```Chrome
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (shella12) [inline README.md]: - [x] Please ☺️  add details to your Getting Started section of the README to reflect your project

File: README.md [hunk 5]
```diff
@@ -1,2 +1,145 @@
...
+I would like to thank a Microverse Full Stack Program
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (shella12) [inline README.md]: - [x] The link to your `MIT LICENCE` is broken. Please 🙏  use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,145 @@
...
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (shella12) [inline README.md]: - [x] Please remove _Note 🗒️ _ from the LICENCE section of the README. The `Note` is there in the template to guide you on how you to add a LICENCE to your README. Once you have successfully added the LICENCE you can go ahead and remove it 👍 😃

File: index.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```
Comment (shella12) [inline index.html]: **Project Requirement** 
> - Add a `.html` file that includes a header with the text "Hello Microverse!". Use h1 tag for that.  

🤓 According to project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you should add your `h1` element inside the `header` semantic tag. I suggest always using semantic tags 🏷️ in your HTML projects as semantics tags add accessibility 💯  to your app 😄 .

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @LarryIVC , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of git flow ✅ 
- Good commit history ✅ 

## Required Changes ♻️
_Check the comments under the review._

- [x] While you have made a great 👍  effort to add ➕  a linter folder 📁 and files 📁 to your project 📔 , you need to make sure that you are using the correct names 📛  for the files and folder. Otherwise, GitHub won’t pass your files through linter checks.  
In your PR I can see that you have added your `linters.yml` file into your folder `.github`, but it should be added to the `workflows` folder, and `workflows` should be added to `.github`.  See [this](https://github.com/microverseinc/linters-config/tree/master/html-css) document to learn how to add workflows folder to your repo and optionally you can also add `.hintrc` and `stylelintrc` files to check and correct linter errors locally (see the document for more details)

Screenshot 2023-02-15 at 10 54 59 AM


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 8]
```diff
@@ -1,2 +1,218 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HelloMicroverse] <a name="about-project"></a>
+
+> This project is about a excercise to understand Microverse tools
```
Comment (Shisui6) [inline README.md]: Please remove these comments from your readme file🙏. These comments are there to guide you on how to personalize the readme template and should not be present in your final readme. Kindly remove all lines which have a comments that start like this

File: README.md [hunk 9]
```diff
@@ -1,2 +1,218 @@
...
+  </ul>
+</details>
+
+<details>
+  <summary>Tool</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/">Client PC - Web Browser</a></li>
+  </ul>
+</details>
+
```
Comment (Shisui6) [inline README.md]: Great job updating your built-with section as suggested by the previous reviewer👍. However, this section still needs a little bit of work. Please remove all comments and update this section to look similar to the screenshot below:

image

- Please remove the `visual studio code editor` from the built with header
- Please move `HTML/CSS` down into the bullet points as seen in the screenshot above👆

File: README.md [hunk 10]
```diff
@@ -1,2 +1,218 @@
...
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Tool used on Microverse
+
+- **[Visual Studio Code]**
+- **[GitBash]**
+- **[HTML]**
+- **[CSS]**
```
Comment (Shisui6) [inline README.md]: As the previous reviewer suggested, kindly update this section to represent a few features of the hello microverse project🙏. Some ideas could be:
- Uses Github actions and linters
- Simple webpage with basic HTML and CSS

File: README.md [hunk 11]
```diff
@@ -1,2 +1,218 @@
...
+```
+### Deployment
+
+You can deploy this project using a browser:
+
+
+Example:
+
+```Chrome index.html
+```
```
Comment (Shisui6) [inline README.md]: [Optional] You can delete this section as it is irrelevant to this project👍. Please note that in future projects where you are required to test and deploy, you will need to have these sections with relevant instructions

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @LarryIVC , 

# Requires Changes 🍀
Great work on making the changes requested by a previous reviewer 👍
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 - Descriptive PR title and summary ✅
 - Follows Github flow ✅
 - Good commit messages ✅

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

_Check the comments under the review._

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 8 above)
Comment (LarryIVC) [reply README.md]: Are the comments the lines that start whit ">" char?

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @LarryIVC , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ...  Green linters
✅ ... Good readme file

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**